WordPress-iOS-Shared/Core/WPFontManager.m

```diff
@@ -172,7 +172,7 @@ + (UIFont *)merriweatherBoldFontOfSize:(CGFloat)size
 + (void)dynamicallyLoadFontResourceNamed:(NSString *)name
 {
     NSString *resourceName = [NSString stringWithFormat:@"%@/%@", kBundle, name];
-    NSURL *url = [[NSBundle mainBundle] URLForResource:resourceName withExtension:@"ttf"];
+    NSURL *url = [[NSBundle bundleForClass:[WPFontManager class]] URLForResource:resourceName withExtension:@"ttf"];
```

Comment:
PS: in static methods `self` refers to the class, and `self.class == self`.  So what I'm saying is that `self` is enough, and `self.class` is sort of redundant.